### DISCUSSION TITLE
How can i connect the Kratos to Salome ?

### DISCUSSION THREAD

**Discussion by jhhan9090** [KratosMultiphysics/Kratos#11682] [General]
Recently, I learned that Kratos has high flexibility and a useful platform.
Therefore, I'm trying to use it for my research.

For visualization, I'd like to use the Salome due to it being free and light to operate.
Thankfully, The plugin code can be found on this page.
(https://github.com/KratosMultiphysics/KratosSalomePlugin/tree/master)

It looked very easy to use.
clone the code and make the system variable.
However, it didn't work properly with some errors.
The plugin functions on Salome didn't work even after I followed the instructions. 

does anyone have an idea about it?

![image](https://github.com/KratosMultiphysics/Kratos/assets/76766267/62d6bf7e-7cca-4669-a635-19eafc6e62b2)

**Comment by jhhan9090** [comment]
I tested it with two kinds of Salome version (9.9 and 9.11)

**Comment by rubenzorrilla** [comment]
@philbucher

**Comment by philbucher** [comment]
hi @jhhan9090 
I did not try the plugin with later versions of salome. I think the failing call can be commented, if my memory serves me correctly, then this was only a safety check. Seems like it no longer work due to restructurings in salome. I would remove the function `__CheckSalomeIsInitialized` and check if it works then

On another note, this plugin is not receiving updates and more, as Kratos can now read MED-files (mesh format used by Code_Aster, writen by salome). For this, the [MedApplication](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/MedApplication) is required. I recommend to check it out. In fact, I am looking for testers right now, so let me know if you are interested :)
